step.yml

```diff
@@ -21,9 +29,8 @@ inputs:
   - enable_debug: "false"
     opts:
       category: Debug
-      title: "Enable debug mode"
-      description: |
-        Enables debug mode. If this input is active, the log of the step will contain more information for easier debugging.
+      title: "Enable debug mode?"
```

Comment:
I'd suggest calling this simply `Debug mode`.

Reply:
It's okay to me!